### PR TITLE
new 404 page and small style fixes

### PR DESCRIPTION
**Pull request by Franz-Kafka**
- added new styled 404 page
- fixed some css

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
-    letter-spacing: -1px;
-  }
-</style>
-
-<div class="container">
-  <h1>404</h1>
-
-  <p><strong>Page not found :(</strong></p>
-  <p>The requested page could not be found.</p>
+<div class="pagehead bg{{ '1,2,3,4,5,6' | split: ',' | sample }}">
```
Comment (Mause) [inline 404.html:8]: Does this randomly choose a new background at build time? Why would we want this?

**Review by Mause — CHANGES_REQUESTED** [review]

File: 404.html (hunk 1 above)
Comment (Franz-Kafka) [reply 404.html:8]: Jonathan would like to have different backgrounds so we thought of random backgrounds. But we could also define each page a specific background. 
I can update.

Comment (Mause) [reply 404.html]: I'm just saying, they're not really random - it would only ever update when we rebuild the website?

**Comment by szarnyasg** [comment]
Thanks! We can tackle the randomness question later.

**Pull request merged**